src/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>News API</title>
+</head>
+<body>
+	<main id="main" class="main">
+		<nav id="nav" class="nav"></nav>
+		<section id="section" class="section"></section>
```

Comment:
I'm not sure that we have a reason for declaring IDs for `main`, `nav` and `section` since we don't use them as a selectors in CSS